ably/export_test.go

```diff
@@ -1,5 +1,13 @@
 package ably
 
-var BuildPath = func(p *PaginatedResource, base, rel string) (string, error) {
+func (p *PaginatedResource) BuildPath(base, rel string) (string, error) {
 	return p.buildPath(base, rel)
 }
+
+func (opts *ClientOptions) RestURL() string {
+	return opts.restURL()
+}
+
+func (opts *ClientOptions) RealtimeURL() string {
+	return opts.RealtimeURL()
+}
```

Comment:
On a side note, this file has been puzzling me for a bit. Isn't this going to be available outside the package anyway?